airflow/www/views.py

```diff
@@ -334,11 +334,20 @@ def task_stats(self, session=None):
         DagRun = models.DagRun
         Dag = models.DagModel
 
-        filter_dag_ids = appbuilder.sm.get_accessible_dag_ids()
+        allowed_dag_ids = set(appbuilder.sm.get_accessible_dag_ids())
 
-        payload = {}
-        if not filter_dag_ids:
-            return
+        if not allowed_dag_ids:
+            return Response('Forbidden', 403)
```

Comment:
Not sure this should be a 403 -- what happens when there are no DAGs in an install (or you don't have permission to any)? Would it make more sense to just return an empty payload instead?

Reply:
Good point. The original statement actually generated a 500 but i will switch it to an empty payload